_layouts/post.html

```diff
@@ -0,0 +1,176 @@
+                  <li><a href="#" target="_blank">Tokens and Tokenomics</a></li>
+                  <li><a href="#" target="_blank">Minting your own (fungible) tokens</a></li>
+                  <li><a href="#" target="_blank">Minting NFTs</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link"><img src="/img/side-nav/document.png" alt="">Smart Contracts</a>
+                <span class="icon"><i class="arrow_carrot-down"></i></span>
+                <ul class="list-unstyled dropdown_nav">
+                  <li><a href="#" target="_blank">Fundamentals: EUTxO</a></li>
+                  <li><a href="#" target="_blank">The Plutus Platform</a></li>
+                  <li><a href="#" target="_blank">Making Sense of Marlowe</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link"><img src="/img/side-nav/content.png" alt="">Videos</a>
+                <span class="icon"><i class="arrow_carrot-down"></i></span>
+                <ul class="list-unstyled dropdown_nav">
+                  <li><a href="https://www.youtube.com/watch?v=ev75v-9LU5I" target="_blank">Making Sense of Blockchain</a></li>
+                  <li><a href="https://www.youtube.com/watch?v=sM0_V53_kGo" target="_blank">The Cardano Revolution</a></li>
+                  <li><a href="https://www.youtube.com/watch?v=mD1kAmP8K-M" target="_blank">Interview with Dr Mihaela Ulieru</a></li>
+                  <li><a href="https://www.youtube.com/watch?v=-k8E_xfLxnI" target="_blank">Submitting a Catalyst Proposal</a></li>
+                </ul>
+              </li>
+            </ul>
+            
+            <!-- <ul class="list-unstyled nav-sidebar bottom_nav">
+              <li class="nav-item">
+                <a href="#" class="nav-link"><img src="../../img/side-nav/edit.png" alt="">English </a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link"><img src="../../img/side-nav/users.png" alt="">Sign In <i class="arrow_right"></i></a>
+              </li>
+            </ul> -->
+          </div>
+        </aside>
+      </div>
+      <div class="col-lg-7 col-md-8 doc-middle-content">
+        <div id="post" class="shortcode_info documentation_info">
+          <article class="shortcode_text">
+            <div class="shortcode_title">
+              <!-- <a class="btn" href="#">Learn</a> -->
+              <h1>{{ page.title }}</h1>
+            </div>
+
+            {{ content | replace: '<img ', '<img class="img-fluid" ' }}
+
+            <footer>
+              <div class="border_bottom"></div>
+              <div class="row feedback_link">
+                <div class="col-lg-6">
+                  <h6><i class="icon_mail_alt"></i>Still stuck? <a href="#" data-toggle="modal" data-target="#exampleModal3">How can we help?</a></h6>
```

Comment:
Had changed this to be a mailto: https://github.com/LovelaceAcademy/Learn/blob/310c596f0186ad784d966a9859e7b7d7c47822f5/index.html#L660